envoy/tests/legacy/common.py

```diff
@@ -0,0 +1,36 @@
+import os
+
+import pytest
+
+from datadog_checks.dev import get_docker_hostname
+
+from ..common import ENVOY_LEGACY
+
+FLAVOR = os.getenv('FLAVOR', 'api_v3')
+
+HOST = get_docker_hostname()
+PORT = '8001'
+INSTANCES = {
+    'main': {'stats_url': 'http://{}:{}/stats'.format(HOST, PORT)},
+    'included_metrics': {
+        'stats_url': 'http://{}:{}/stats'.format(HOST, PORT),
+        'metric_whitelist': [r'envoy\.cluster\..*'],
+    },
+    'excluded_metrics': {
+        'stats_url': 'http://{}:{}/stats'.format(HOST, PORT),
+        'metric_blacklist': [r'envoy\.cluster\..*'],
+    },
+    'included_excluded_metrics': {
+        'stats_url': 'http://{}:{}/stats'.format(HOST, PORT),
+        'included_metrics': [r'envoy\.cluster\.'],
+        'excluded_metrics': [r'envoy\.cluster\.out\.'],
+    },
+    'collect_server_info': {
```

Comment:
I see that `collect_server_info` is still kept for tests, but it's no longer in your [conf.yaml.example](https://github.com/DataDog/integrations-core/blob/4babc2c98a3e162a7da15121f4cd52f40e1a1d62/envoy/datadog_checks/envoy/data/conf.yaml.example). It's still used in the legacy implementation, but it's still in the [legacy implementation](https://github.com/DataDog/integrations-core/blob/4babc2c98a3e162a7da15121f4cd52f40e1a1d62/envoy/datadog_checks/envoy/envoy.py#L52). Should this be included as a config option?

Reply:
see above comment, the readme links to the legacy implementation conf.yaml.example.

Reply:
We'll need to collect the version still